src/components/Register.jsx

```diff
@@ -43,77 +44,80 @@ function Register() {
-          <legend>Information</legend>
-          <div className="form-data">
-            <label htmlFor="name">Name</label>
-            <input
-              type="text"
-              id="name"
-              name="name"
-              onChange={onChange}
-              value={user.name}
-              placeholder="John Doe"
-            />
-          </div>
+        <form onSubmit={submitForm}>
+          <fieldset>
+            <legend>Vos informations</legend>
+            <div className="form-data">
+              <label htmlFor="name">Nom</label>
+              <input
+                type="text"
+                id="name"
+                name="name"
+                onChange={onChange}
+                value={user.name}
+                placeholder="John Doe"
+              />
+            </div>
 
-          <div className="form-data">
-            <label htmlFor="username">Username</label>
-            <input
-              type="text"
-              id="username"
-              name="username"
-              onChange={onChange}
-              value={user.username}
-              placeholder="johndoe"
-            />
-          </div>
+            <div className="form-data">
+              <label htmlFor="username">Nom d&apos;utilisateur</label>
+              <input
+                type="text"
+                id="username"
+                name="username"
+                onChange={onChange}
+                value={user.username}
+                placeholder="johndoe"
+              />
+            </div>
 
-          <div className="form-data">
-            <label htmlFor="email">Email</label>
-            <input
-              type="email"
-              id="email"
-              name="email"
-              onChange={onChange}
-              value={user.email}
-              placeholder="johndoe@gmail.com"
-            />
-          </div>
+            <div className="form-data">
+              <label htmlFor="email">Email</label>
+              <input
+                type="email"
+                id="email"
+                name="email"
+                onChange={onChange}
+                value={user.email}
+                placeholder="johndoe@gmail.com"
+              />
+            </div>
 
-          <div className="form-data">
-            <label htmlFor="password">Password</label>
-            <input
-              type="password"
-              id="password"
-              name="password"
-              onChange={onChange}
-              value={user.password}
-              placeholder="********"
-            />
-          </div>
+            <div className="form-data">
+              <label htmlFor="password">Mot de passe</label>
+              <input
+                type="password"
+                id="password"
+                name="password"
+                onChange={onChange}
+                value={user.password}
+                placeholder="********"
+              />
+            </div>
```

Comment:
missing a confirmation for the password. even if you don't use it, show it ;)

Reply:
We added an input for the password confirmation. We will use it in the next sprint.